### PR TITLE
Remove overlay bg

### PR DESCRIPTION
**Pull request by devin5**

### CODE REVIEW

File: src/components/vispage/ExtendedRadial.jsx [hunk 1]
```diff
@@ -50,129 +51,33 @@ function ExtendedRadial({
   const hollowSize = getHollowSize(krValues.names, 50);
   const radialMargin = 25 - (krValues.names.length * 3.2) / 2;
 
-  const [visState] = useState({
```
Comment (danchapman) [inline src/components/vispage/ExtendedRadial.jsx:53]: what's this big deletion about?

Comment (danchapman) [reply src/components/vispage/ExtendedRadial.jsx:53]: nvm, I see you moved them into an options block

File: src/hooks/utils/reduceStylesObject.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { cloneDeep, clone, reduce } from "lodash";
+
+const makeReduceStylesObject = (params, ...args) => {
```
Comment (danchapman) [inline src/hooks/utils/reduceStylesObject.js:3]: Did you mean to include this file? I don't see it referenced in these changes

**Review by danchapman — COMMENTED** [review]
One question about a file that was added. I probably would have put all your clean-up refactor in a separate PR, but it's not terrible that it's in here with your main focus.

File: src/hooks/utils/reduceStylesObject.js (hunk 2 above)
Comment (devin5) [reply src/hooks/utils/reduceStylesObject.js:3]: this was deleted by mistake and added back in here

**Pull request merged**